components/ui/PriceInEuro.tsx

```diff
@@ -0,0 +1,36 @@
+import { Currency } from "@/app/guide/shared/ui/Currency";
+import { Tooltip } from "./Tooltip";
+
+export interface PriceInEuroProps {
+  eur?: number;
```

Comment:
How would this be used?

Reply:
This will be used to show a euro amount as a converted USD amount.

It is print friendly (expand to `XXX€ (YYY$ rate Z.ZZZ on YYYY-MM-DD)`) and in the UI the details are shown in a tooltip.

![image](https://github.com/hinosxz/tax-helper/assets/245501/bea30e5f-f50a-4f9e-b372-5c7df3d56e9a)

Reply:
To make sure the prices displayed matches the ones computed by the tax rule.
There should be no difference, but sometimes rounding is not consistent...